schema/base/landCover.yaml

```diff
@@ -0,0 +1,83 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Cover Schema
+description: representation of the Earth's natural surfaces 
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    description: classification of surface, can correspond to the 'natural' tag from OpenStreetMap
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - subType
+      - minZoom
+      - maxZoom
+    properties:
+      subType:
+        description: type of surface represented
+        type: string
+        enum:
+          - barren
+          - crop
+          - forest
+          - grass
+          - mangrove
+          - moss
+          - shrub
+          - snow
+          - urban
+          - wetland
+      minZoom:
+        description: recommended minimum zoom according to bing tile zoom levels
+        type: number
+        minimum: 1
+        maximum: 24
+      maxZoom:
+        description: recommended maximum zoom according to bing tile zoom levels
```

Comment:
"Bing tile" seems to be an inside-Meta term for just standard slippy map tile zooms... Is there a better name?

Reply:
i thought "bing tile" was a pretty standard term, but yes it corresponds to ["slippy map" tile names](https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames) and the [original Google](https://www.maptiler.com/google-maps-coordinates-tile-bounds-projection/#3/-2.54/50.02) tile spec.

ill update accrodingly.